corehq/apps/fixtures/fixturegenerators.py

```diff
@@ -45,42 +56,86 @@ class ItemListsProvider(FixtureProvider):
 
     def __call__(self, restore_state):
         restore_user = restore_state.restore_user
-
-        all_types = {t._id: t for t in FixtureDataType.by_domain(restore_user.domain)}
-        global_types = {id: t for id, t in all_types.items() if t.is_global}
-
+        global_types = {}
+        user_types = {}
+        for data_type in FixtureDataType.by_domain(restore_user.domain):
+            if data_type.is_global:
+                global_types[data_type._id] = data_type
+            else:
+                user_types[data_type._id] = data_type
+        items = self.get_global_items(global_types, restore_state)
+        items.extend(self.get_user_items(user_types, restore_user))
+        return items
+
+    def get_global_items(self, global_types, restore_state):
+        restore_user = restore_state.restore_user
+        user_id = restore_user.user_id
+        domain = restore_user.domain
+        db = get_blob_db()
+        if not restore_state.overwrite_cache:
+            global_id = GLOBAL_USER_ID.encode('utf-8')
+            b_user_id = user_id.encode('utf-8')
+            try:
+                data = db.get(domain, FIXTURE_BUCKET).read()
```

Comment:
This uses the same key that was used in #16904 and #16951, which means it may return previously cached data that is likely outdated and definitely contains the wrong `user_id` for most users.

It's pretty easy to delete the `FIXTURE_BUCKET` in the blob db (needs to be run on each env before this is deployed):
```py
from corehq.blobs import get_blob_db
FIXTURE_BUCKET = 'domain-fixtures'
db = get_blob_db()
db.delete(bucket=FIXTURE_BUCKET)
```

Reply:
should this get added to a migration?

Reply:
6a74915c3f197ef197a34514c7ff313ac0a68d2f